### PR TITLE
font-mononoki

### PR DESCRIPTION
**Pull request by heliostatic**
##### Instructions

- Look for and complete the section relevant to your submission. Delete the others, including these `Instructions`.
- `{{cask_file}}` represents the cask file you’re submitting/editing (if applicable).
- If there’s a checkbox you can’t complete for any reason, that’s OK. Just explain in detail why you weren’t able to do so.

#### Editing an existing cask

- [ ] Commit message includes cask’s name (and new version, if applicable).
- [ ] `brew cask audit --download {{cask_file}}` is error-free.
- [ ] `brew cask style --fix {{cask_file}}` left no offenses.

#### Adding a new cask

- [x] Checked there aren’t open [pull requests](https://github.com/caskroom/homebrew-fonts/pulls) for the same cask.
- [x] Checked there aren’t closed [issues](https://github.com/caskroom/homebrew-fonts/issues) where that cask was already refused.
- [x] When naming the cask, followed the [reference](https://github.com/caskroom/homebrew-fonts/blob/master/CONTRIBUTING.md#naming-font-casks).
- [x] Commit message includes cask’s name.
- [x] `brew cask audit --download {{cask_file}}` is error-free.
- [x] `brew cask style --fix {{cask_file}}` left no offenses.
- [x] `brew cask install {{cask_file}}` worked successfully.
- [x] `brew cask uninstall {{cask_file}}` worked successfully.

### CODE REVIEW

File: Casks/font-mononoki.rb [hunk 1]
```diff
@@ -0,0 +1,14 @@
+cask 'font-mononoki' do
+  version :latest
+  sha256 :no_check
+
+  url 'https://github.com/madmalik/mononoki/raw/master/export/mononoki.zip'
```
Comment (vitorgalvao) [inline Casks/font-mononoki.rb]: Please add an [`appcast`](https://github.com/caskroom/homebrew-cask/blob/master/doc/cask_language_reference/stanzas/appcast.md).

Comment (adidalal) [inline Casks/font-mononoki.rb]: No appcast at the moment, but will merge anyway. Cask can be updated later with proper versioning (as it's (:latest right now, anyway)

**Comment by adamyonk** [comment]
Can this be reopened now since the mononoki repo is using GH releases?

**Comment by adidalal** [comment]
Yes, feel free to submit an updated PR!

**Pull request merged**